### ISSUE TITLE
multi loss layers

### ISSUE THREAD

**Issue by giladdiv** [fchollet/keras#4218]
Please make sure that the boxes below are checked before you submit your issue. Thank you!
- [x] Check that you are up-to-date with the master branch of Keras. You can update with:
  pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps
- [x] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
  pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps
- [x] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

**Comment by giladdiv** [comment]
![image](https://cloud.githubusercontent.com/assets/23100778/19801350/8da02084-9d07-11e6-9d05-8258ffb22476.png)
Hello,
I want to build this net architecture in keras but i stil don't understand how to implement this multi loss layer. The idea is to have softmax loss functions for every one of the N classes which will be trained only when the input is of the correct class.
what is the right way to do it?
Thank you

**Comment by dieuwkehupkes** [comment]
Perhaps you could write a custom lossfunction in objectives.py that returns 0 if the classes do not match? Similar to #549?
